types/define-properties/define-properties-tests.ts

```diff
@@ -18,30 +18,29 @@ define(object);
 define(null, {});
 
 define(object, {
-	getFoo() {
+	getFoo() { // $ExpectType () => string
 		this; // $ExpectType any
 
 		return __classPrivateFieldGet(this, object_foo);
 	}
 });
 
 define(object, {
-	foo: 'any'
+	foo: 'any' // $ExpectType string
 }, {
 	foo: () => (object as any).foo !== 'any'
 });
 
-// $ExpectError
 define(object, {
-	foo: 'any'
+	foo: 'any' // $ExpectType string
 }, {
 	foo: () => (object as any).foo !== 'any',
-	bar: () => { throw new Error(); },
+	bar: () => { throw new Error(); }, // $ExpectError
```

Comment:
Previously, this&nbsp;error would’ve&nbsp;been&nbsp;reported for&nbsp;the&nbsp;map&nbsp;parameter, which&nbsp;wasn’t&nbsp;all&nbsp;that&nbsp;helpful.